projects/plugins/crm/src/js/state/automations-admin/actions.ts

```diff
@@ -10,7 +10,13 @@ export type AutomationsAction =
 	| HydrateWorkflowsAction
 	| ActivateWorkflowAction
 	| DeactivateWorkflowAction
-	| SetAttributeAction;
+	| SetAttributeAction
+	| SelectWorkflowAction
+	| DeselectWorkflowAction
+	| SelectAllWorkflowsAction
+	| DeselectAllWorkflowsAction
+	| ActivateAllSelectedWorkflowsAction
```

Comment:
Is this meant to be `ActivateSelectedWorkflowsAction` ?

```suggestion
	| ActivateSelectedWorkflowsAction
```

Reply:
Yes, that's correct.